src/content/1.7/basics/installation/distribution.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Create your own distribution
+menuTitle: Create your own distribution
+weight: 35
+---
+
+For creating your own distribution, you must [create a release][create-release].
```

Comment:
I feel this needs a little more details. Why would someone create a distribution? Shouldn't we put this inside a new section (after installation) called "Deployment"? Or does it make sense that this is under "Basics" whereas this is an advanced feature?

Reply:
Fixed